### DISCUSSION TITLE
How to define wait_time in config-users?

### DISCUSSION THREAD

**Discussion by evanchaoli** [locustio/locust#2848] [Q&A]
I want to config a user's wait_wait to be `constant_pacing(5)` in a user config JSON file, but the following JSON doesn't work:

```
  {
    "user_class_name": "MyApiUser",
    "wait_time": "constant_pacing(5)"
  }
```

Is there a way to do that?

**Comment by cyberw** [comment]
config-users only supports setting plain values (strings or ints). Perhaps this feature could be added...

As a workaround, you could define your own wait_time method that reads a static value from the User class and set that value using config-users (or from a regular custom argument)

**Reply by evanchaoli** [reply]
@cyberw I am using the workaround as you mentioned. I defined a custom parameter "calls_per_sec", then I convert it to "wait_wait". But my follow up question is that, where is the best place to do the translation? I am now calling the translate function in every task function, which leads to a lot of duplicate code.
